src/frontend/src/lib/components/hero/Hero.svelte

```diff
@@ -1,12 +1,16 @@
 <script lang="ts">
+	import { quintOut } from 'svelte/easing';
+	import { slide } from 'svelte/transition';
 	import HeroContent from '$lib/components/hero/HeroContent.svelte';
 
 	export let usdTotal = false;
 	export let summary = false;
-	export let actions = true;
 	export let back = false;
 </script>
 
-<article class="main relative flex flex-col items-center rounded-lg pb-6 pt-10">
-	<HeroContent {usdTotal} {summary} {actions} {back} />
+<article
+	class="relative flex flex-col items-center rounded-lg pb-6"
+	transition:slide={{ delay: 0, duration: 250, easing: quintOut, axis: 'y' }}
```

Comment:
Not for this PR, just thinking out at loud, maybe we can create a constant for `{ delay: 0, duration: 250, easing: quintOut, axis: 'y' }`

Reply:
YES! THANK YOU (Micheal Scott style)

Reply:
Done with PR:

- PR #2807